R/fit.R

```diff
@@ -1502,11 +1502,11 @@ CmdStanMCMC <- R6::R6Class(
 #'   `"log_lik"`. This argument is passed to the [`$draws()`][fit-method-draws]
 #'   method.
 #' @param r_eff (multiple options) How to handle the `r_eff` argument for `loo()`:
-#'   * `TRUE` (the default) will automatically call [loo::relative_eff.array()]
-#'   to compute the `r_eff` argument to pass to [loo::loo.array()].
-#'   * `FALSE` or `NULL` will avoid computing `r_eff` (which can sometimes be slow),
-#'   but the reported ESS and MCSE estimates can be over-optimistic if the
-#'   posterior draws are not (near) independent.
+#'   * `TRUE` will call [loo::relative_eff.array()] to compute the `r_eff`
+#'   argument to pass to [loo::loo.array()].
```

Comment:
Add something like:
`r_eff` measures the amount of autocorrelation in MCMC draws, and is used to compute more accurate ESS and MCSE estimates for pointwise and total elpds.